### PR TITLE
Adding Chime SDK FAQs and updated Readme to include new demos

### PR DESCRIPTION
**Pull request by vidya-mohan**
**Issue #:** 
<no issue>

**Description of changes:**
Added new guide addressing frequently asked questions.
In addition linked 2 new demos that were published from readme.

**Testing**

1. Have you successfully run `npm run build:release` locally?
YES
2. How did you test these changes?
Tested .md file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: guides/06_FAQs.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+# Amazon Chime Javascript SDK - Frequently Asked Questions
```
Comment (simmkyu) [inline guides/06_FAQs.md:1]: => Amazon Chime SDK for JavaScript

File: guides/06_FAQs.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+## General questions
+
+### What is the Amazon Chime SDK?
+
+The [Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk)is a set of real-time communications components that developers can use to quickly add audio calling, video calling, and screen sharing capabilities to their own web or mobile applications. Developers can leverage the same communication infrastructure and services that power Amazon Chime, an online meetings service from AWS, and deliver engaging experiences in their applications.
```
Comment (simmkyu) [inline guides/06_FAQs.md:7]: Add a space between `)` and `is`

File: guides/06_FAQs.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+## General questions
+
+### What is the Amazon Chime SDK?
+
+The [Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk)is a set of real-time communications components that developers can use to quickly add audio calling, video calling, and screen sharing capabilities to their own web or mobile applications. Developers can leverage the same communication infrastructure and services that power Amazon Chime, an online meetings service from AWS, and deliver engaging experiences in their applications. 
+
+There are three client SDKs that can be used by applications based on the use case. Each of these repos have walkthroughs, and links to demos and sample code:
+
+Amazon Chime SDK for JavaScript https://github.com/aws/amazon-chime-sdk-js
```
Comment (simmkyu) [inline guides/06_FAQs.md:11]: Fix the list layout. I'm previewing using this URL:
https://github.com/aws/amazon-chime-sdk-js/blob/2c6fd4a58a51087660e02381989a5fb834c7ece4/guides/06_FAQs.md

File: guides/06_FAQs.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+Amazon Chime SDK for iOS https://github.com/aws/amazon-chime-sdk-ios
+Amazon Chime SDK for Android https://github.com/aws/amazon-chime-sdk-android
+
+### How much does the Amazon Chime SDK cost?
+
+Amazon Chime uses utility-based pricing.  Meetings are billed based on attendee minutes (from the time an attendee joins and then leaves a meeting), in 0.1 minute increments.
+
+Example 1: A 10 minute meeting with 2 people attending for the whole meeting costs 20 attendee minutes.
```
Comment (ltrung) [inline guides/06_FAQs.md:19]: This does not seem to show up as separate line in the result file.

File: guides/06_FAQs.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+Example 2: A 10 minute meeting, with 2 people attending for the whole time, and 1 person joining for the last 5 minutes, costs 25 attendee minutes.
+
+Billing starts when the attendee joins the meeting and leaves the meeting.  Calling [CreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html) or [BatchCreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_BatchCreateAttendee.html) does not initiate billing, rather it just creates the join token to enable an attendee to join.
+
+Additional information on pricing is available on the [Amazon Chime pricing page](https://aws.amazon.com/chime/pricing/#Chime_SDK_). If you have more questions, contact [Customer support](https://pages.awscloud.com/GLOBAL-aware-GC-Amazon-Chime-SDK-2020-reg.html).
+
+### I have a question not addressed in this FAQ. What is my next step?
+
+If you have feature requests or feedback, please fill out the [customer questionnaire](https://pages.awscloud.com/GLOBAL-aware-GC-Amazon-Chime-SDK-2020-reg.html)for us to reach out to you.
```
Comment (simmkyu) [inline guides/06_FAQs.md:28]: Add a space between `)` and `for`

File: guides/06_FAQs.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+### How many attendees can join an Amazon Chime SDK meeting? Can this limit be raised?
+
+Amazon Chime SDK limits are defined [here](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). The service supports up to 250 attendees and up to 16 video participants in a meeting (video limit is enforced separately). An attendee is considered active unless it has been explicitly removed using DeleteAttendee API call. Attendee limits cannot be changed. 
+
+If your use case requires more than 250 attendees, consider using a [broadcasting solution](https://quip-amazon.com/e8N3AnXbdH1w/Chime-Javascript-SDK-Frequently-Asked-Questions-DRAFT#UDQ9CAA1oPv).
```
Comment (simmkyu) [inline guides/06_FAQs.md:73]: Replace the Quip doc with the public GitHub URL

File: guides/06_FAQs.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+### Can I schedule Amazon Chime SDK meetings ahead of time?
+
+The Amazon Chime SDK does not support scheduling meetings ahead of time. The moment [CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html#API_CreateMeeting_RequestBody) is invoked and the meeting is created, the auto end policies will kick in and if no one joins the meeting, it will close within 5 minutes. The application can create the meeting when drawing closer to the meeting start time or when the the first attendee is ready to join.
```
Comment (simmkyu) [inline guides/06_FAQs.md:77]: Remove `#API_CreateMeeting_RequestBody`

File: guides/06_FAQs.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+Applications can get logs from Chime SDK by passing instances of Logger when instantiating [the MeetingSession object](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSession.ts#L14). Amazon Chime SDK has some default implementations of logger that your application can use, such as [ConsoleLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/ConsoleLogger.ts) which logs into the browser console, [MeetingSessionPOSTLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MeetingSessionPOSTLogger.ts) which logs in [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) and [MultiLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MultiLogger.ts) which logs in multiple destinations.
+
+On mobile phones such as iPhone you can also do some advance debugging by connecting the phone to a Mac such as highlighted in this [article](https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html).
```
Comment (simmkyu) [inline guides/06_FAQs.md:133]: Sorry, I gave you this link. This Safari doc is deprecated. I can't find an offical doc on how to debug the mobile Safari using the desktop. You can remove this sentence for now and re-add it with examples.

File: guides/06_FAQs.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+## Networking
+
+### What are the subnets and ports used by an application using the AWS SDK?
+
+An application which calls Amazon Chime APIs via the [AWS SDK](https://aws.amazon.com/sdk-for-node-js/), like [CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html), connects to the global service endpoint in the us-east-1 region: [service.chime.aws.amazon.com](http://service.chime.aws.amazon.com/), or [service-fips.chime.aws.amazon.com](http://service.chime.aws.amazon.com/) with TCP:443.
```
Comment (simmkyu) [inline guides/06_FAQs.md:143]: nit: You have linked "AWS SDK for Node.js" but a customer can be using AWS SDK in any language. How about adding https://aws.amazon.com/tools ?

File: guides/06_FAQs.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+
+### What are the subnets and ports used by a client running the Amazon Chime SDK?
+
+A client running the Amazon Chime SDK connects to the Amazon Chime service via URIs (never direct IP addresses) that are part of the *.chime.aws domain.  The URIs resolve to IP addresses in the CHIME_MEETINGS subnets, as published in the [AWS ip-ranges.json](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html), which is currently a single subnet: 99.77.128.0/18.
+
+The Amazon Chime SDK will use TCP:443 for https and websocket connectivity, and UDP:3478 for media.  If UDP:3478 is blocked, the SDK falls back to TCP, however the video experience will be poor, and audio will use a region specific subnet assigned by [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/).
+
+### Can a client running the Amazon Chime SDK connect to the service via a proxy?
+
+The Amazon Chime SDK has URL[rewriter hooks](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionurls.html#urlrewriter) that the application builder can use to munge the URLs to traverse unauthenticated proxies.  When traversing a proxy, all media is sent over TCP which impacts the user experience.
```
Comment (simmkyu) [inline guides/06_FAQs.md:155]: Add a space between `URL` and `rewriter`

File: guides/06_FAQs.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+
+Amazon Chime JS SDK applications do not support content sharing on mobile browsers because a key dependency, WebRTC's [getDisplayMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getDisplayMedia) is not available on these browsers. Clients joining from these devices can still view content shared by other participants in the call.
+
+### Will Amazon Chime JS SDK meetings work in Chrome and Firefox in iOS?
+
+At present, Amazon Chime JS SDK is supported on Safari/iOS only. Other browsers are missing a key dependency, [GetUserMedia](https://bugs.webkit.org/show_bug.cgi?id=208667).
+
+### When loading a webpage in iOS WebView, my application gets a TypeError: undefined is not an object (evaluating 'navigator.mediaDevices.addEventListener'), what could be the issue ?
+
+This is because Safari embedded [web view does not support WebRTC](https://forums.developer.apple.com/thread/88052). The mitigation is to use the Amazon Chime SDK for iOS to build a native application: https://github.com/aws/amazon-chime-sdk-ios
```
Comment (simmkyu) [inline guides/06_FAQs.md:173]: https://forums.developer.apple.com/thread/88052  - the link is broken (403) as of 6:24 PM PST. Probably, a temporary error?

File: guides/06_FAQs.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+
+### I see a slate which says “Your client does not support hardware acceleration” when I enable video on my device?
+
+This error indicates that the device you are using does not support hardware acceleration decoding. However this does not impact the ability of this user to participate in Amazon Chime SDK meetings as the device can render and transmit VP8 streams to other parties in the call. Specifically for Chrome, you will need to enable Unified Plan support by setting this [flag](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enableunifiedplanforchromiumbasedbrowsers) to true or enable [Simulcast](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enablesimulcastforunifiedplanchromiumbasedbrowsers). 
+In some cases, H.264 may be missing from the initial SDP offer ( [tracking Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1047994)) causing this slate to appear. Rejoining the meeting can fix the videos to be rendered once again.
```
Comment (simmkyu) [inline guides/06_FAQs.md:195]: Remove a space between `(` and `[tracking`

File: guides/06_FAQs.md [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+// Stop the meeting session (audio and video)
+meetingSession.audioVideo.stop();
+```
+
+### My clients are unable to successfully join audio calls from Safari, they get a “failed to get audio device for constraints null: Type error”, what could be the issue?
+
+This error message is an indication that the browser application did not successfully acquire the media stream for audio or video from the device before the meeting starts. Application has not passed in the right device Id to the `chooseVideoInputDevice` API. In this case you will see the following entry in the log where an empty string after `chooseVideoInputDevice`:
+
+```
+`[Info] 2020-06-18T17:43:55.380Z [INFO] VLR - API/DefaultDeviceController/chooseVideoInputDevice "" -> "PermissionDeniedByBrowser"`
```
Comment (simmkyu) [inline guides/06_FAQs.md:235]: Remove backtick characters ``

File: guides/06_FAQs.md (hunk 6 above)
Comment (vidya-mohan) [reply guides/06_FAQs.md:73]: good one

File: guides/06_FAQs.md (hunk 11 above)
Comment (vidya-mohan) [reply guides/06_FAQs.md:173]: yeah this used to work. will fix it later

Comment (vidya-mohan) [reply guides/06_FAQs.md:176]: It works now Kyu - with no change from us

File: guides/06_FAQs.md [hunk 14]
```diff
@@ -0,0 +1,242 @@
+# Amazon Chime SDK for Javascript - Frequently Asked Questions
```
Comment (simmkyu) [inline guides/06_FAQs.md:1]: Javascript => JavaScript

File: guides/06_FAQs.md [hunk 15]
```diff
@@ -0,0 +1,242 @@
...
+
+### How can I learn about interruptions to the Amazon Chime service? 
+
+You can be notified about Amazon Chime service interruptions at the https://status.aws.amazon.com/ website. In addition you can set up [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) events via the [AWS Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/) for the Amazon Chime service. You can find the Amazon Chime SLA on this [webpage](https://aws.amazon.com/chime/sla/).
+
+### Is the Amazon Chime SDK supported on mobile browsers?
+
+Amazon Chime SDK for JavaScript is supported on certain mobile browsers, specifically Chrome/Android and Safari/iOS. Developers can also build native mobile applications using the following SDKs (this option allows for meetings to continue when applications are sent to the background).
+
+Amazon Chime SDK for iOS https://github.com/aws/amazon-chime-sdk-ios
```
Comment (simmkyu) [inline guides/06_FAQs.md:51]: Fix the layout

**Pull request merged**